task/04-arrays-tasks.js

```diff
@@ -469,7 +502,7 @@ function getIntervalArray(start, end) {
  *   [ 1, 1, 2, 2, 3, 3, 4, 4] => [ 1, 2, 3, 4]
  */
 function distinct(arr) {
-   throw new Error('Not implemented');
+   return arr.filter((x, i) => arr.indexOf(x) === i);
```

Comment:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator or `callback`. You will make less mistakes.